### ISSUE TITLE
Create a local database that is accessible with API

### ISSUE THREAD

**Issue by TheGlorySaint** [serverpod/serverpod#1378]
I have a project in my mind, but can't find the relevant information in the documentation.

I'd like to create a Server and Client App. The Server App is only useable from a desktop device. The Client App can every Device (Desktop, Tablet, Phone) install.

My Idea is, that the Server App only has the purpose to serve a database that is only accessible with API from the local network _(two devices that run the client app and connect with the IP-Address (for example 192.168.178.21.0:8443) from the Server Device)_.

Does my idea work with serverpod?

**Comment by Isakdl** [comment]
Yes there is nothing that would stop you from doing this. This is merely a hosting question, I.E. how you want to run the server.
